packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -2882,6 +2883,88 @@ describe("BnsConnection", () => {
     connection.disconnect();
   });
 
+  it("can register/transfer and update a username for an empty account", async () => {
+    pendingWithoutBnsd();
+    const connection = await BnsConnection.establish(bnsdTendermintUrl);
+    const chainId = connection.chainId();
+
+    const { profile, faucet, walletId } = await userProfileWithFaucet(chainId);
+    const emptyPath = HdPaths.iov(666);
+    const user = await profile.createIdentity(walletId, chainId, emptyPath);
+    const userAddress = identityToAddress(user);
+    const username = `user${Math.random()}*iov`;
+
+    const userAccount = await connection.getAccount({ address: userAddress });
+    if (userAccount && userAccount.balance.length) {
```

Comment:
Is `userAccount.balance` still empty when the account exists on chain? Or to rephrase the question: can this test pass multiple times on the same chain?

Reply:
Yes, the NFTs don't appear in the balance.